types/leveldown/leveldown-tests.ts

```diff
@@ -4,19 +4,19 @@ import LevelDOWN, { Bytes } from 'leveldown';
 const a = new LevelDOWN("/tmp/db");
 const b = LevelDOWN("/tmp/db");
 
-const down = new LevelDOWN("/tmp/db");
+const db = new LevelDOWN("/tmp/db");
 
-down.open(() => {
-  down.put("key", "value", (err: Error | undefined) => { });
-  down.put(Buffer.from([1]), "value", { something: true }, (err: Error | undefined) => { });
+db.open(() => {
+  db.put("key", "value", (err: Error | undefined) => { });
+  db.put(Buffer.from([1]), "value", { something: true }, (err: Error | undefined) => { });
 
-  down.get("key", (err: Error | undefined) => { });
-  down.get(Buffer.from([1]), { something: true }, (err: Error | undefined, value: Bytes) => { });
+  db.get("key", (err: Error | undefined) => { });
+  db.get(Buffer.from([1]), { something: true }, (err: Error | undefined, value: Bytes) => { });
 
-  down.close((err: Error | undefined) => {
-    // do nothing
-  });
+  db.close((err: Error | undefined) => { });
 });
 
-down.destroy("/tmp/db", (err: Error | undefined) => { });
-down.repair("/tmp/db", (err: Error | undefined) => { });
+db.clear((err: Error | undefined) => { });
+
+LevelDOWN.destroy("/tmp/db", (err: Error | undefined) => { });
```

Comment:
This was changed in comparison of previous PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40028.

Previous code compiles without error, because `LevelDOWN` extends `AbstractLevelDOWN`, which extends `AbstractOptions`, where each key has `any` type https://github.com/DefinitelyTyped/DefinitelyTyped/blob/13e6949a025a03d92596652a2bbd64448ed71382/types/abstract-leveldown/index.d.ts#L8-L10